packages/editor/src/editor-ui/plugin-toolbar/plugin-toolbar.tsx

```diff
@@ -20,11 +24,19 @@ export function PluginToolbar({
   contentControls,
   pluginSettings,
   pluginControls,
+  pluginTooltipText,
   className,
   noWhiteShadow,
 }: PluginToolbarProps) {
   const pluginStrings = useEditorStrings().plugins
 
+  const pluginTooltip = pluginTooltipText ? (
+    <span className="serlo-tooltip-trigger ml-[-8px]">
+      <EditorTooltip text={pluginTooltipText} />
+      <FaIcon icon={faCircleQuestion} className="mr-2" />
+    </span>
+  ) : null
+
```

Comment:
Minor, but, consistency again: All the other conditional renders in this file are inline JSX conditionals. Introducing this approach as well makes it harder, for the next dev working on this file, to keep track of what's going on. Please try to keep only one approach. Any approach is fine with me, constants for render blocks like you did here, inline JSX conditionals like this file had already, `function render*()` functions underneath the return statement like in older serlo code, whatever - as long as it's only one approach in one file.

Reply:
Done